lib/src/theme/text.dart

```diff
@@ -0,0 +1,26 @@
+class AppString {
+  static const safe = "Safe";
+  static const bump = "Bump";
+  static const tracking = "Tracking";
+  static const tools = "Tools";
+  static const educational = "Educational";
+  static const resources = "Resources";
+  static const community = "Community";
+  static const support = "Support";
+  static const appointment = "Appointment";
+  static const scheduler = "Scheduler";
+  static const welcome = "Welcome";
+  static const skip = "Skip";
+  static const next = "Next";
+  static const welcomeToSafeBump =
+      "Welcome to SafeBump, your trusted partner for a safe and healthy pregnancy journey.";
+  static const theAppWillProvideTracking =
+      "The app will provide tracking tools to help users monitor their pregnancy and postpartum progress, including weight tracking, contraction timing, and breastfeeding tracker.";
+  static const theAppWillProvideEducational =
+      "The app will provide educational resources to help users learn about maternal health, including articles, videos, and podcasts.";
+  static const theAppWillProvideACommunity =
+      "The app will provide a community support feature to help users connect with other users, share experiences, and receive support.";
+  static const theAppWillAllowUser =
+      "The app will allow users to schedule and manage their appointments with healthcare providers, such as doctors and midwives.";
+  static const getStarted = "Get Started";
+}
```

Comment:
You should use [flutter_localizations](https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization) to make it easier to make multi-language feature in the future